build/build.yml

```diff
@@ -227,15 +209,3 @@ steps:
   - script: dotnet nuget push $(Build.ArtifactStagingDirectory)\$(BuildConfiguration)\*.nupkg --source artifacts-credprovider --api-key az --skip-duplicate
     displayName: dotnet push
     condition: eq(variables['PushNupkg'], 'true')
-
-  - task: ManifestGeneratorTask@0
-    displayName: 'SBOM Generator'
-    inputs:
-      BuildDropPath: '$(Build.ArtifactStagingDirectory)\$(BuildConfiguration)\'
-      Verbosity: Verbose
-
-  - task: PublishBuildArtifacts@1
```

Comment:
Same comment here, let's switch this to PublishPipelineArtifact and keep for the PR/CI case

Reply:
Added this back!